scripts/setup-new-module

```diff
@@ -0,0 +1,352 @@
+#!/bin/bash
```

Comment:
It looks like almost all of the other scripts are in python (except weirdly the minor-version-bump, but its fairly simple).  Is there a reason to do this in bash instead?  Personally I tend to find longer bash scripts to be somewhat more difficult to read.

Reply:
No specific reason, agreed it's more readable in python. Converted to python